### PR TITLE
homework for day 2

### PR DESCRIPTION
**Pull request by alanoudssr**
DAY 2 HW:

- How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
     
2

- Was there anything that you struggled with?;

Checking for the last two letters of a word in the Pluralizer section (e.g. pluralizing words that end with "y" differs depending on the letter before y). I solved this with a slice() function, but it took some research, and I'm not 100% sure that it's the cleanest way to solve this problem.  

- Is there anything that you'd like some further information on?;

No, thank you.

- Roughly how long did it take?

Around 2-3 hours

### CODE REVIEW

**Comment by alanoudssr** [comment]
DAY 3 HW:

- How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;

1

- Was there anything that you struggled with?;

No

- Is there anything that you'd like some further information on?;

No, thank you.

- Roughly how long did it take?
 
An hour

File: Alanoud_AlRasheed/week_01/day_02/conditionals_lab/main.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+}
+
+//Serge Says
+
+var userMessage = "     ";
+var upperCaseMsg = userMessage.toUpperCase();
+
+if(userMessage.endsWith("?")){
+    console.log("Sure");
+} else if(userMessage.trim()===""){
```
Comment (mfalthaw) [inline Alanoud_AlRasheed/week_01/day_02/conditionals_lab/main.js]: Nice!

File: Alanoud_AlRasheed/week_01/day_02/conditionals_lab/main.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+} else if(userMessage===upperCaseMsg){
+    console.log("Woah, chill out!");
+} else {
+    console.log("Whatever");
+}
+
+//The Pluralizer
+
+var word = "life";
+var lastTwoLetters = word.slice(-2);
```
Comment (mfalthaw) [inline Alanoud_AlRasheed/week_01/day_02/conditionals_lab/main.js]: Perfect use of `.slice()`, one of the handiest functions in JS!

File: Alanoud_AlRasheed/week_01/day_02/conditionals_lab/main.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    console.log(word+"es");
+} else if (word.endsWith("y") && lastTwoLetters!=="ay" && lastTwoLetters!=="ey" && lastTwoLetters!=="iy" && lastTwoLetters!=="uy" && lastTwoLetters!=="oy"){
+    console.log(word.slice(0,-1)+"ies");
+} else if (word.endsWith("f")) {
+    console.log(word.slice(0,-1)+"ves");
+} else if (lastTwoLetters==="fe") {
+    console.log(word.slice(0,-2)+"ves");
+} else {
+    console.log(word+"s");
+}
```
Comment (mfalthaw) [inline Alanoud_AlRasheed/week_01/day_02/conditionals_lab/main.js]: I gotta say, the most sophisticated _pluralizer_ I've seen today!

File: Alanoud_AlRasheed/week_01/day_03/Functions/main.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+var numThree = 2;
+var numFour = 4;
+var percentResult = percentOf(numThree, numFour);
+console.log(numThree+" is "+percentResult+"% of "+numFour+".");
+
+// - Write a function called `areaOfCircle` that will take one argument (the radius), calculate the area based on that, and return the result. It should also log a string like `"The area for a circle with radius 2 is 12.566370614359172."`
+// - Bonus: Round the result so there are only two digits after the decimal.
+
+function areaOfCircle(radius) {
+    var area = 3.14159265359 * (radius * radius);
```
Comment (mfalthaw) [inline Alanoud_AlRasheed/week_01/day_03/Functions/main.js]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/PI

**Pull request merged**